docs/docs/20-quickstart.md

```diff
@@ -60,6 +60,12 @@ for Docker Desktop for Mac OS only. You can follow
 [these instructions](https://docs.orbstack.dev/kubernetes/) to enable its
 built-in Kubernetes support.
 
+To ensure that your Kubernetes cluster is up and running, use the following command:
+
+```shell
+orb start k8s
+```
+
```

Comment:
I am supportive of this kind of improvement, but let's spike it out into a separate PR. If we start being more detailed about how to start it, we probably need to be more detailed about stopping/deleting it, with applicable disclaimers, etc. It just begins to seem like its own discrete issue.